src/main/java/frc/robot/maps/subsystems/ElevatorMap.java

```diff
@@ -55,38 +58,49 @@ public double getValue(ElevatorPresets preset) {
     public final SmartMotorController leftMotor;
     public final SmartMotorController rightMotor;
     public final IEncoder encoder;
-    public final double conversionRate;
     public final ElevatorPresetValues elevatorPreset;
     public final ValueRange hardLimits;
     public final ValueRange softLimits;
+    public final ProfiledPIDController pid;
+    public final ElevatorFeedforward feedForward;
 
     public ElevatorMap() {
-        this(new SmartMotorController(), new SmartMotorController(), new MockEncoder(), 1, new ElevatorPresetValues(),
-                new ValueRange(0, 0), new ValueRange(0, 0));
+        this(new SmartMotorController(), new SmartMotorController(), new MockEncoder(), new ElevatorPresetValues(),
+                new ValueRange(0, 0), new ValueRange(0, 0), new ProfiledPIDController(0, 0, 0, new Constraints(0, 0)),
+                new ElevatorFeedforward(0, 0, 0));
     }
 
     public ElevatorMap(SmartMotorController leftMotor, SmartMotorController rightMotor, IEncoder encoder,
-            double conversionRate, ElevatorPresetValues elevatorPreset, ValueRange hardLimits, ValueRange softLimits) {
+            ElevatorPresetValues elevatorPreset, ValueRange hardLimits, ValueRange softLimits,
+            ProfiledPIDController pid, ElevatorFeedforward feedForward) {
         this.leftMotor = leftMotor;
         this.rightMotor = rightMotor;
```

Comment:
The Elevator subsystem does not need to know that there are two motors. We should be configuring one of them as a follower in the robot map.